clients/roscpp/src/libros/rosout_appender.cpp

```diff
@@ -102,7 +103,13 @@ void ROSOutAppender::log(::ros::console::Level level, const char* str, const cha
   msg->file = file;
   msg->function = function;
   msg->line = line;
-  this_node::getAdvertisedTopics(msg->topics);
+  
+  // check parameter server/cache for omit_topics flag
+  ros::param::getCached("/rosclient/omit_topics", omit_topics_);
```

Comment:
There's already precedent for a small number of  global parameters which are consumed by the client libraries rather than user code. I'm thinking in particular of `/rosdistro`, `/rosversion`, `/use_sim_time`, `/tcp_keepalive`, and possibly others.

So I think the right pattern here would be a non-namespaced topic that is verbose enough that it's unlikely to cause a collision. Perhaps something like `/rosout_omit_topics`?

Also, regardless of what is chosen, please put a comment on each of these changes alerting to the parallel code in the other location.

Reply:
won't a topic name like that cause confusion with `/rosout/omit_topics` from the other PR? perhaps something that specifies that the topics list is no longer being generated, like `/rosout_disable_topics_fetch`?

Reply:
I went ahead and made the changes, if you prefer `/rosout_omit_topics` just let me know and I can change it again!